{"uid":"604fb6be13c4724","name":"CliKintoneTest-110 Should return the error message when exporting the record with incorrect --guest-space-id","fullName":"features/record/export.feature#CliKintoneTest-110 Should return the error message when exporting the record with incorrect --guest-space-id","historyId":"146185e4cc3a546d9f83e20aad590398:d41d8cd98f00b204e9800998ecf8427e","time":{"start":1734426094590,"stop":1734426095164,"duration":574},"description":"","status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"description":"","status":"passed","steps":[{"name":"Given Load app ID of the app \"app_in_guest_space\" as env var: \"APP_ID\"","time":{"start":1734426094591,"stop":1734426094591,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"And Load app token of the app \"app_in_guest_space\" with exact permissions \"view\" as env var: \"API_TOKEN\"","time":{"start":1734426094591,"stop":1734426094591,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"When I run the command with args \"record export --base-url $$TEST_KINTONE_BASE_URL --app $APP_ID --api-token $API_TOKEN --guest-space-id 9999999999999999999\"","time":{"start":1734426094591,"stop":1734426095164,"duration":573},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Then I should get the exit code is non-zero","time":{"start":1734426095164,"stop":1734426095164,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"And The output error message should match with the pattern: \"\\[403] \\[CB_NO02] No privilege to proceed\"","time":{"start":1734426095164,"stop":1734426095164,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":5,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"host","value":"fv-az711-959"},{"name":"language","value":"javascript"},{"name":"framework","value":"cucumberjs"},{"name":"thread","value":"2125"},{"name":"feature","value":"cli-kintone export command"},{"name":"suite","value":"CliKintoneTest-110 Should return the error message when exporting the record with incorrect --guest-space-id"},{"name":"tag","value":"@isolated"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":0,"broken":0,"skipped":0,"passed":3,"unknown":1,"total":4},"items":[{"uid":"2858420010cc4bf5","reportUrl":"https://kintone.github.io/cli-kintone/6758//#testresult/2858420010cc4bf5","status":"passed","time":{"start":1734425848847,"stop":1734425849471,"duration":624}},{"uid":"1501fca1dd9ed850","reportUrl":"https://kintone.github.io/cli-kintone/6757//#testresult/1501fca1dd9ed850","status":"passed","time":{"start":1734421177590,"stop":1734421178313,"duration":723}},{"uid":"e65a69a5638f0084","reportUrl":"https://kintone.github.io/cli-kintone/6670//#testresult/e65a69a5638f0084","status":"unknown","statusDetails":"The test doesn't have an implementation.","time":{"start":1733124270466,"stop":1733124270467,"duration":1}}]},"tags":["@isolated"]},"source":"604fb6be13c4724.json","parameterValues":[]}